diff --git a/tools/jdk/default_java_toolchain.bzl b/tools/jdk/default_java_toolchain.bzl
index 183dd55b10..21e5af89f8 100644
--- a/tools/jdk/default_java_toolchain.bzl
+++ b/tools/jdk/default_java_toolchain.bzl
@@ -88,7 +88,7 @@ DEFAULT_TOOLCHAIN_CONFIGURATION = dict(
     ] + JDK9_JVM_OPTS,
     turbine_jvm_opts = [
         # Turbine is not a worker and parallel GC is faster for short-lived programs.
-        "-XX:+UseParallelOldGC",
+        # "-XX:+UseParallelOldGC",
     ],
     java_runtime = "@bazel_tools//tools/jdk:remote_jdk11",
 )
@@ -123,7 +123,7 @@ PREBUILT_TOOLCHAIN_CONFIGURATION = dict(
     ] + JDK9_JVM_OPTS,
     turbine_jvm_opts = [
         # Turbine is not a worker and parallel GC is faster for short-lived programs.
-        "-XX:+UseParallelOldGC",
+        # "-XX:+UseParallelOldGC",
     ],
     ijar = ["@bazel_tools//tools/jdk:ijar_prebuilt_binary"],
     singlejar = ["@bazel_tools//tools/jdk:prebuilt_singlejar"],
@@ -138,7 +138,7 @@ NONPREBUILT_TOOLCHAIN_CONFIGURATION = dict(
     ] + JDK9_JVM_OPTS,
     turbine_jvm_opts = [
         # Turbine is not a worker and parallel GC is faster for short-lived programs.
-        "-XX:+UseParallelOldGC",
+        # "-XX:+UseParallelOldGC",
     ],
     ijar = ["@remote_java_tools//:ijar_cc_binary"],
     singlejar = ["@remote_java_tools//:singlejar_cc_bin"],