diff --git a/MODULE.bazel b/MODULE.bazel
index 84e08db938..883b22d845 100644
--- a/MODULE.bazel
+++ b/MODULE.bazel
@@ -99,11 +99,11 @@ maven.install(
         "com.google.code.findbugs:jsr305:3.0.2",
         "com.google.code.gson:gson:2.9.0",
         "com.google.code.java-allocation-instrumenter:java-allocation-instrumenter:3.3.4",
-        "com.google.errorprone:error_prone_annotation:2.22.0",
-        "com.google.errorprone:error_prone_annotations:2.22.0",
-        "com.google.errorprone:error_prone_check_api:2.22.0",
-        "com.google.errorprone:error_prone_core:2.22.0",
-        "com.google.errorprone:error_prone_type_annotations:2.22.0",
+        "com.google.errorprone:error_prone_annotation:2.36.0",
+        "com.google.errorprone:error_prone_annotations:2.36.0",
+        "com.google.errorprone:error_prone_check_api:2.36.0",
+        "com.google.errorprone:error_prone_core:2.36.0",
+        "com.google.errorprone:error_prone_type_annotations:2.36.0",
         "com.google.flogger:flogger:0.5.1",
         "com.google.flogger:flogger-system-backend:0.5.1",
         "com.google.flogger:google-extensions:0.5.1",
