diff --git a/MODULE.bazel b/MODULE.bazel
index 14fc324..2905f3f 100755
--- a/MODULE.bazel
+++ b/MODULE.bazel
@@ -368,9 +368,9 @@ rbe_preconfig(
     toolchain = "ubuntu2004",
 )
 
-list_source_repository = use_repo_rule("//src/test/shell/bazel:list_source_repository.bzl", "list_source_repository")
+# list_source_repository = use_repo_rule("//src/test/shell/bazel:list_source_repository.bzl", "list_source_repository")
 
-list_source_repository(name = "local_bazel_source_list")
+# list_source_repository(name = "local_bazel_source_list")
 
 winsdk_configure = use_repo_rule("//src/main/res:winsdk_configure.bzl", "winsdk_configure")
 
